src/NuGetPackager/PackageCreator.cs

```diff
@@ -27,7 +27,7 @@ public PackageCreator(string packagingFolderFullPath, string nugetsFolderFullPat
                 { "authors", () => "NServiceBus Ltd" },
                 { "owners", () => "NServiceBus Ltd" },
                 { "licenseUrl", () => "http://particular.net/LicenseAgreement" },
-                { "projectUrl", () => "http://particular.net/" },
+                { "projectUrl", () => $"https://docs.particular.net/nuget/{projectName}/" },
```

Comment:
Not at the moment, but should be after merging this https://github.com/Particular/DocsEngine/pull/316/files, as I understand?